janitor/functions.py

```diff
@@ -4045,8 +4104,8 @@ def expand_grid(
         If no dataframe exists, all inputs
         in others will be combined to create a dataframe.
     :returns: A pandas dataframe of all combinations of name value pairs.
-    :raises: TypeError if others is not a dictionary
-    :raises: KeyError if there is a dataframe and no key is provided.
+    :raises TypeError: if others is not a dictionary
+    :raises KeyError: if there is a dataframe and no key is provided.
```

Comment:
Just an exmaple that ``raises`` clauses should include the type of error raised between colons. This is pretty cool because it automatically puts a link to the Pythons type errors in the docs.

After making this change, the docs show links to [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) and [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError)


![Screen Shot 2020-10-01 at 08 46 29](https://user-images.githubusercontent.com/24984410/94817432-9d263e00-03c2-11eb-9955-bd25d68fcc69.png)

Reply:
I think I like this one. Helpful for users. How do you create the hyperlink?

Reply:
It's built in and automatic 😄 As long as the ``raises`` clause includes the error *within* the colons.

Reply:
@loganthomas , this is really cool. Definitely have to play with this in my next PR. If I get stuck, I'll reach out to you.